docs/api.md

```diff
@@ -65,6 +65,40 @@ when a VSP is closed will result in an error.
 **Registering a ticket is a two step process. The VSP will not add a ticket to
 its voting wallets unless both of these calls have succeeded.**
 
+#### Step Zero (optional)
+
+Set an alternate signing address for a ticket. The ticket must be valid and will
+be transmitted to the network if not found. If set, the vsp will check that a
```

Comment:
> If set, the vsp will check that a signature is good for this address and fallback to the commitment address if not.

I guess this is leftover from when alt sig could be updated, because it doesnt seem to make sense any more

Reply:
I think this is still true. It will check if set and fallback to the commitment address.